br/pkg/pdutil/pd.go

```diff
@@ -841,7 +849,12 @@ func (p *PdController) ResetTS(ctx context.Context, ts uint64) error {
 	})
 	var err error
 	for _, addr := range p.addrs {
-		_, e := pdRequest(ctx, addr, resetTSPrefix, p.cli, http.MethodPost, bytes.NewBuffer(reqData))
+		code, _, e := pdRequestWithCode(ctx, addr, resetTSPrefix, p.cli, http.MethodPost, bytes.NewBuffer(reqData))
+		// for pd version <= 6.2, if the given ts < current ts of pd, pd returns StatusForbidden.
+		// it's not an error for br
+		if code == http.StatusForbidden {
```

Comment:
Maybe we should test restore to cluster with 3 pds

Reply:
pd replica doesn't matter here